code/_onclick/hud/screen_objects/screen_objects.dm

```diff
@@ -764,12 +764,17 @@
 /obj/screen/arrow/proc/add_hud(mob/living/carbon/tracker_input, atom/target_input)
 	if(!tracker_input?.client)
 		return
-
 	tracker = tracker_input
 	target = target_input
 	tracker.client.screen += src
```

Comment:
probably should harddel protect this reference specifically

Reply:
There was literraly a runtime here splinter. So it is hapenning.
"what probably happens is the arrow getting destroyed and it not being removed from the client screen." 
I have another PR for that